### ISSUE TITLE
Remove `role="group"` from `<address>` elements.

### ISSUE THREAD

**Issue by tempertemper** [tempertemper/tempertemper.net#394]
`role="group"` currently prevents `<address>` elements getting an implicit `role="contentinfo"`, which is better for screen reader users. This implicit role is a bug and has been removed in Safari Technology Preview, so `role="group"` should be removed from all `<address>` elements once the fix lands in Safari.
